### ISSUE TITLE
Publish custom files with Lambda package?

### ISSUE THREAD

**Issue by JamieCressey** [serverless/serverless#439]
I'm looking at migrating a couple of standalone Lambda functions and corrosponding API gateway endpoints to Serverless, one of which I currently package with a keyfile which is used to authenticate to a 3rd party API.

I've tested my new Serverless function locally with the custom file within the /back/modules/<module>/<function> directory and it works fine. Is there a way to tell Serverless to package custom files up with the function when it's published to Lambda?

Thanks!

**Comment by joostfarla** [comment]
By default, Serverless zips the whole `/back/modules` directory for each function, including any extra files in there. Did you try to deploy the function? If yes, what was the exact result?

**Comment by JamieCressey** [comment]
Within `back/modules/<module>/<function>` I have a file called key.pem. Within my Lambda code I call it as `./key.pem` (which works in my standalone Lambda module), however when I deploy my Serverless version and test the uploaded function I receive:

`{
    "errorMessage": "ENOENT, open './key.pem'",
    "errorType": "Error",
    "stackTrace": []
}`

I've downloaded the Serverless package from the Lambda UI and can see it contains the full `modules` directory structure, so I've updated my Lambda code to source the file from `modules/<module>/<function>/key.pem` which appears to have worked, however my local tests `serverless function run` only run when I'm in the `back/` directory. Should I call these custom files with a relative or absolute path?
